UserDocs.md

```diff
@@ -38,6 +38,8 @@ Writes the in-memory object `src` to the path `dest` in S3.
 Transparently serializes some object types.
 See [Serialization](#serialization) for details.
 
+`dest` should not end in `/`. Objects that end in `/` are treated specially by some s3 tools in a way that is potentially dangerous, so it's best to avoid them.
```

Comment:
not the right place to document this - because `he.put()` automatically forbids it